Translation/en/Text/regex.txt

```diff
@@ -1,4 +1,4 @@
-r:"^女性([０-９0-9]+)人(／|/)男性([０-９0-9]+)人$"=$1 Female / $2 Male
+r:"^女性([０-９0-9]+)人(?:／|/)男性([０-９0-9]+)人$"=$1 Female / $2 Male
```

Comment:
Another reason to use named groups. But `[／/]` should also work fine and be a bit cleaner.
```suggestion
r:"^女性([０-９0-9]+)人[／/]男性([０-９0-9]+)人$"=$1 Female / $2 Male
```